python/friesian/dev/test/run-example-tests-tf2.6.0.sh

```diff
@@ -62,9 +62,53 @@ python ../../example/wnd/train/wnd_train_recsys.py \
 now=$(date "+%s")
 time2=$((now - start))
 
+echo "#3 start example test for deepfm train"
+
+#timer
+start=$(date "+%s")
+if [ -d data/input_2tower ]; then
+  echo "data/input_2tower already exists"
+else
+  wget -nv $FTP_URI/analytics-zoo-data/input_2tower.tar.gz -P data
+  tar -xvzf data/input_2tower.tar.gz -C data
+fi
+
+python ../../example/deep_fm/deepFM_train.py \
+    --executor_cores 4 \
+    --executor_memory 10g \
```

Comment:
these won't take effect for local mode right?

Reply:
yes, The default value is enough for running

Reply:
Then remove the cores and executor memory